docs/STEPS.md

```diff
@@ -1158,6 +1160,39 @@ Support
 |:-----:|:--------------:|:----------:|
 |   ✔   |        ✔       |      ✔     |
 
+## Set timeout \<seconds>
+
+> \* Set timeout "1"
+
+This step overwrites the property `driver_implicit_timeout` for a number of steps, that support it. For now, those steps are:
+* [Assert \<by> = \<by_value> exists](#assert-by--by_value-exists)
+* [Assert \<by> = \<by_value> does not exist](#assert-by--by_value-does-not-exist)
+* [Save placeholder \<placeholder> from attribute \<attribute> of \<by> = \<by_value>]
```

Comment:
`[Save placeholder \<placeholder> from attribute \<attribute> of \<by> = \<by_value>](#save-placeholder-placeholder-from-attribute-attribute-of-by--by_value)`
I missed the right part of the link. Pls also double check the other links